OPERATOR.rst

```diff
@@ -211,28 +211,17 @@ To do a promotion:
 
 #. ::
 
-      git checkout prod
+      git checkout develop
+      git checkout -b promotions/<yyyy-mm-dd>
+      git push github promotions/<yyyy-mm-dd>
 
-#. ::
-
-      git merge develop --no-ff
-
-   Use the default merge commit message, which should be ``Merge branch 'develop' into prod``.
-
-#. Examine the changes to ``prod`` since the last promotion to determine if a reindex
-   is necessary.
+#. File a PR on github from the new promotions branch which targets ``prod``.
```

Comment:
```suggestion
#. File a PR on Github from the new promotions branch which targets ``prod``.
```

Reply:
Thanks for spotting that. I think this review probably shouldn't have been an approval though.